server.js

```diff
@@ -33,6 +36,9 @@ mongoose
   .then(() => console.log(`Yay - Connection to cloud database established!`))
   .catch((err) => console.log('[ERROR] DB Connection failed', err));
 
+// Routes
+app.use('/users', usersRouter);
+
```

Comment:
Are we going to have just one callback for `/users`? Right now it can be weird that the records functions are being called inside a function called `usersRouter` It makes more sense if then inside that route the records depend on each user, but right now that is not the case.

Reply:
I don't understand... can you explain what you mean? 
i haven't set the "/" yet, cause its about the routes in this branch.

Reply:
and if we change like we discussed we have more anyway.

Reply:
Right now every endpoint you are using is being called after `/users`  because inside usersRouter you all calling all the other middlewares